mswindows/osgeo4w/mklibs.sh

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+if [ "$CI" ] ; then
+	DUMPBIN_PATH=$(cygpath -ua "C:/Program Files (x86)\Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/HostX64/x64/")
```

Comment:
What is diference between `DUMPBIN_PATH` and `VCPATH`? I thought that `VCPATH` is basically the path to `dumpbin` binary.